spec/factories/cohorts.rb

```diff
@@ -6,5 +6,17 @@
     initialize_with do
       Cohort.find_by(start_year:) || new(**attributes)
     end
+
+    trait :current do
+      start_year { Date.current.year - (Date.current.month < 9 ? 1 : 0) }
+    end
+
+    trait :next do
+      start_year { Date.current.year + (Date.current.month < 9 ? 0 : 1) }
+    end
```

Comment:
Should we just be explicit about years when building cohorts - this feels like it could lead to unpredictable behaviour.

Also I think this logic could be expressed a little more simply, eg:

```
    trait :next do
      start_year { Date.current.month < 9 ? Date.current.year : Date.current.year.succ
    end
```